### PR TITLE
Added my github graduation info

### PR DESCRIPTION
**Pull request by victoriateo**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [ ] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/victoriateo/victoriateo.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Victoria Teo
+institution: National University of Singapore
+profile_pic: photo6174763929457240098.jpg
+quote: "Hard work beats talent when talent doesn't work hard! And also, follow the KISS principle - Keep It Simple, Stupid!"
```
Comment (Mohammed-Shoaib) [inline _data/victoriateo/victoriateo.md:5]: Quote must **not** be longer than 100 characters, yours is **117** characters.

**Comment by victoriateo** [comment]
Hey there, I have resized the image to 1:1 aspect ratio and shortened my quote. Please see if it's okay! Thanks!

File: _data/victoriateo/victoriateo.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Victoria Teo
+institution: National University of Singapore
+profile_pic: photo6174763929457240098.jpg
```
Comment (Mohammed-Shoaib) [inline _data/victoriateo/victoriateo.md:4]: Incorrect file name, please do one of the following:
1. Change `photo6174763929457240098.jpg` to `photo6174763929457240098 (1).jpg` in the markdown file.
2. Rename `photo6174763929457240098 (1).jpg` to `photo6174763929457240098.jpg`.
I'd suggest you rename the file!

**Comment by victoriateo** [comment]
Sorry my bad, I made the necessary change, hope it works now!

**Comment by victoriateo** [comment]
Yay thank you so much! All the best to you both too! Can't wait to receive the swag.

**Pull request merged**